tslint.json

```diff
@@ -23,7 +23,8 @@
         "@hypertrace/graphql-client",
         "@hypertrace/test-utils",
         "@hypertrace/observability",
-        "ng-mocks"
+        "ng-mocks",
+        "highlight.js"
```

Comment:
this wouldn't be required if you declare highlight.js as a peer dependency

Reply:
add peer in components -> package.json

Reply:
Done, thanks